<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1" colspan="2">
     <widget class="QGroupBox" name="gb_tomo">
      <property name="title">
       <string>Tomogram</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_contrast">
         <property name="text">
          <string>Contrast</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QSlider" name="slider_contrast_min">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSlider" name="slider_contrast_max">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lbl_denoise_param">
         <property name="toolTip">
          <string>Number of slices in both directions used for averaging</string>
         </property>
         <property name="text">
          <string>Neighbor slices</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinb_denoise_param">
         <property name="toolTip">
          <string>Number of slices in both directions used for averaging</string>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="button_denoise">
         <property name="text">
          <string>Denoise</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QPushButton" name="button_load_tomo">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QGroupBox" name="gb_lmap">
      <property name="title">
       <string>Label map</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1" colspan="2">
        <widget class="QSlider" name="slider_opacity">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_opacity">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QPushButton" name="button_load_lmap">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <layout class="QVBoxLayout" name="layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
